serviceerror/convert_test.go

```diff
@@ -98,6 +98,37 @@ func TestToStatus_NotServiceError(t *testing.T) {
 	require.Len(t, st1.Details(), 0)
 }
 
+func TestToStatus_MultiOp(t *testing.T) {
```

Comment:
What about `FromStatus` so Go SDK users can see these errors?

Reply:
👍 I didn't realize that the Go SDK is calling `FromStatus`.

Reply:
:+1: This library is (transitively) our most used library by users :grin: